custom_components/tuya_local/devices/wifi_smart_garage_door.yaml

```diff
@@ -0,0 +1,65 @@
+name: Garage door
+products:
+  - id: keyftvxappyrem5a
+    name: WiFi Smart Garage Door 
+primary_entity:
+  entity: cover
+  class: garage
+  dps:
+    - id: 1
+      type: boolean
+      name: control
+      optional: true
+      mapping:
+        - dps_val: true
+          value: open
+        - dps_val: false
+          value: close
+    - id: 101
+      name: open
+      type: boolean
+secondary_entities:
+  - entity: number
+    name: Travel time
+    icon: mdi:timer
+    category: config
+    dps:
+      - id: 102
+        type: integer
+        name: value
+        unit: s
+        range:
+            min: 10
+            max: 120
+        mapping:
+          - step: 1
+  - entity: number
+    name: Timer
+    hidden: true
+    icon: mdi:timer
```

Comment:
When there is a colon in the value, it needs to be quoted to avoid yaml parsing errors.

Reply:
Oops dumb typo